internal/cluster/validator.go

```diff
@@ -528,23 +527,85 @@ func (v *clusterValidator) printNtpServerConfigured(c *clusterPreprocessContext,
 	}
 }
 
-func (v *clusterValidator) isOcsRequirementsSatisfied(c *clusterPreprocessContext) validationStatus {
-	validationStatus := (v.operatorsManager.ValidateOCSRequirements(c.cluster))
-	if validationStatus == "pending" {
-		return ValidationPending
-	} else if validationStatus == "success" {
-		return ValidationSuccess
+// sufficientMemoryAvailableForOperators: A cluster is considered to have a sufficient amount of memory when the amount of memory usable per node is greater
+// than the sum of the minimum memory requirements for each of the operators enabled for the cluster.
+// In a nutshell, a successful validation looks like this:
+//     sum(Node usable memory) > sum (minimum memory required for each enabled operator)
+func (v *clusterValidator) sufficientMemoryAvailableForOperators(c *clusterPreprocessContext) validationStatus {
+
+	opMgr := operators.NewManager(v.log)
```

Comment:
Please use the manager that's created in `main.go`- the OCS validator is stateful and that may in the future have impact on this code.

Reply:
Thanks. Will do.